### ISSUE TITLE
Accessibility Particle

### ISSUE THREAD

**Issue by gioras** [gantry/gantry5#2050]
There are few extensions in the market that will address the font size and contrast compatibilities in WCAG2 As the need to implement those requirements increases, it would be nice to see a particle in Gantry5 that would address those requirements.

Thanks

Best Regards,

Giora

**Comment by N8Solutions** [comment]
I also agree that Accessibility in website design is important. I would like to see this as an option as well. However I don't believe a particle would suffice. I believe something like this would have to be built in to the Gantry 5 Framework. A particle could be created that controls the functionality of the Accessibility options and to allow the website designer to enable that functionality on the front end of the website.

**Comment by N8Solutions** [comment]
I wanted to add that in the absence of an accessibility particle I have been using the widget from https://userway.org/ and have found it to be very useful.
